lib/rbac/filterer.rb

```diff
@@ -543,6 +543,10 @@ def scope_targets(klass, scope, rbac_filters, user, miq_group)
         scope = scope_to_cloud_tenant(scope, user, miq_group)
       end
 
+      if klass.respond_to?(:rbac_scope_for_model)
+        scope = scope.rbac_scope_for_model(user)
```

Comment:
@gtanzillo @Fryguy Most of our filtering has been kept in rbac/filterer - especially when the classes didn't have to handle a bunch of different subclasses / similar cases.

Do you have a preference whether the logic of the filtering lives in rbac or in the model themselves?

Personally prefer keeping the `user.super_admin_user?` logic in rbac, but would like to hear your preferences